docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md

```diff
@@ -94,21 +93,26 @@ Not applicable as this is the initial release of GDC 2.0.
 * Cart:
     * Spinners on the Download Cart and Download Associated Data buttons may be displayed longer than expected. This is a visual issue and does not affect the use of these buttons. <!--SV-2343-->
     * More than 5 GB of files in total may be downloaded at a time via the browser if the user first attempts to download controlled access data without being logged in, then logs in via the information dialog displayed before continuing with the download. <!--SV-2342-->
+    * Using multiple browser tabs with the portal when adding or removing files from the cart may result in the cart not being updated as expected. <!--SV-2412-->
 * Mutation Frequency:
     * Gene/mutation sets created from the tables in Mutation Frequency may contain 0 genes/mutations if the cohort has Available Data filters or Biospecimen filters. <!--SV-2314-->
-    * Attempting to download a TSV of all the mutations in the GDC may result in an error due to the length of time needed to generate the TSV. As a workaround, limit the number of mutations downloaded by using a smaller cohort with the tool. <!--SV-2388-->
+    * Attempting to download a TSV of all the mutations in the GDC may result in an error due to the length of time needed to generate the TSV. As a workaround, limit the number of mutations downloaded to 1.5 million. <!--SV-2388-->
 * Gene Expression Clustering:
     * A spinner overlay can scroll over the entire page if a user scrolls while the overlay is present.
     * The label to the right of the gene expression cluster heatmap gets cut off. 
     * The tool displays "no data" as an error message when trying to plot a project without gene expression data.
+* Quick Search may not display results if the search input is the same as the previous input and entered very quickly. As a workaround, temporarily change the input before reentering the intended search. <!--SV-2410-->
```

Comment:
not loving this phrasing either...

Reply:
"Quick Search may not display results if the the same search input is applied twice quickly. As a workaround, temporarily change the input before reentering the intended search."